packages/rspack-dev-server/package.json

```
@@ -18,8 +18,9 @@
     "./package.json": "./package.json"
   },
   "scripts": {
+    "prepare": "prebundle",
     "build": "tsc -b ./tsconfig.build.json",
```

Use `tsc-alias` to apply alias paths

```suggestion
    "build": "tsc -b ./tsconfig.build.json && tsc-alias -p tsconfig.build.json",
```